cpp/src/transform/row_bit_count.cu

```diff
@@ -484,8 +485,9 @@ std::unique_ptr<column> row_bit_count(table_view const& t,
   CUDF_EXPECTS(info.size() == cols.size(), "Size/info mismatch");
 
   // create output buffer and view
-  auto output = cudf::make_fixed_width_column(
-    data_type{type_id::INT32}, t.num_rows(), mask_state::UNALLOCATED, stream, mr);
+  auto const num_segments = t.num_rows() / segment_length;
```

Comment:
Seems like a failure case here if number of rows isn't evenly divisible by segment size.  Ie, 7 rows, segment size 8.

Reply:
I think this should be `div_rounding_up_safe`